test_data_conversion_scripts/convert_xlsx_tests_csv_lohnst.py

```diff
@@ -18,6 +18,7 @@
     "in_ausbildung",
     "ges_krankenv_zusatzbeitr_satz",
     "ges_pflegev_zusatz_kinderlos",
+    "regulär_beschäftigt",
```

Comment:
I think we don't need this anymore, correct?

Reply:
True